components/GlobalStyles.js

```diff
@@ -27,7 +27,7 @@ export const StyledHeader = styled.header`
   }
 
   p {
-    margin-top: 1px;
+    margin: 10px, 10px, 10px, 10px;
```

Comment:
You could also write:

```suggestion
    margin: 10px;
```

Reply:
Agree! makes sense.